google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -111,7 +111,6 @@ public void actionPerformed(ActionEvent e) {
         if (mySyncStashCheckbox.isVisible()) {
           myWarningLabel.setVisible(!mySyncStashCheckbox.isSelected());
```

Comment:
how testable are these DialogWrapper subclasses?

A few tests to document what the warning and visibility and selected flags should be for each use-case could help people understand what this is up to.